### PR TITLE
ICU-20120 63 BRS, integrate cldr release-34-beta with various fixes

### PR DESCRIPTION
**Pull request by pedberg-icu**
Various data fixes (plus version updates):
- currencies: nl, capitalization of CNY display name
- locales: kl, fixed spelling of Sunday
- units: various locales, fixed abbreviation for fathom
- units: ru, fixed spelling for wide/meter-per-second/few

##### Checklist

- [x] Issue filed at https://unicode-org.atlassian.net :  ICU-20120
- [x] Update PR title to include Issue number
- [x] Issue accepted
- [x] Tests included
- [ ] Documentation is changed or added: N/A

### CODE REVIEW

File: icu4c/source/data/locales/kl.txt [hunk 1]
```diff
@@ -46,7 +46,7 @@ kl{
             dayNames{
                 format{
                     abbreviated{
-                        "sab",
+                        "sap",
```
Comment (sffc) [inline icu4c/source/data/locales/kl.txt]: https://unicode.org/cldr/trac/ticket/11453

File: icu4c/source/data/unit/bn.txt [hunk 2]
```diff
@@ -379,8 +379,8 @@ bn{
             }
             fathom{
                 dnam{"fm"}
-                one{"{0} ftm"}
-                other{"{0} ftm"}
+                one{"{0} fth"}
```
Comment (sffc) [inline icu4c/source/data/unit/bn.txt]: https://unicode.org/cldr/trac/ticket/10623

File: icu4c/source/data/unit/kl.txt [hunk 3]
```diff
@@ -11,8 +11,8 @@ kl{
         length{
             fathom{
                 dnam{"fm"}
-                one{"{0} ftm"}
-                other{"{0} ftm"}
+                one{"{0} fth"}
```
Comment (sffc) [inline icu4c/source/data/unit/kl.txt]: Here and a couple other places: why no change to the dnam?

File: icu4c/source/data/unit/ru.txt [hunk 4]
```diff
@@ -828,7 +828,7 @@ ru{
             }
             meter-per-second{
                 dnam{"метры в секунду"}
-                few{"{0} метра в секундыу"}
+                few{"{0} метра в секунду"}
```
Comment (sffc) [inline icu4c/source/data/unit/ru.txt]: https://unicode.org/cldr/trac/ticket/11445

File: icu4c/source/data/unit/kl.txt (hunk 3 above)
Comment (pedberg-icu) [reply icu4c/source/data/unit/kl.txt]: Not sure, I will file a separate CLDR ticket about that, thanks. But not a blocker for this merge or for ICU 63 generally.

**Comment by jefgen** [comment]
Does this also need to be cherry-picked into the `maint-63` branch, or is the plan to recreate the maint-63 branch off of master once the BRS items are done?

**Comment by sffc** [comment]
It needs a cherry-pick.

**Pull request merged**